Notify/Notify/Notify/Core/Notification.cs

```diff
@@ -28,6 +28,7 @@ public class Notification
         
         public bool IsRenewable => Status == "Expired" && Type != NotificationType.Time;
         public bool IsEditable => Status != "Expired";
+        public bool IsDynamicLocation => Type.Equals(NotificationType.Dynamic);
```

Comment:
I remember we had some issues using the Equals method in this kind of operations..
I'd prefer using the old fashion operator== here.